### PR TITLE
Feature/landingpage

### LINKED ISSUE #26: Connect controlled form to Firebase

### PR DESCRIPTION
**Pull request by asiisii**
## Description
Please include a summary of the change and which issue is fixed. Please also include relevant screenshots of your code being deployed. 
- Connected controlled for to firebase

Fixes # (issue)
- close #26 

## Type of change
Please delete options that are not relevant.
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [x] Refactoring

## Checklist:
- [ ] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have checked my code and corrected any misspellings

### CODE REVIEW

File: src/components/LoginConfig/LoginConfig.js [hunk 1]
```diff
@@ -46,7 +46,17 @@ class LoginConfig extends React.Component {
     auth.signOut()
   }
 
-  
+  changeAuthState() {
```
Comment (rhen92) [inline src/components/LoginConfig/LoginConfig.js:49]: I like how you are changing the state based on if there is a user or not.

File: src/firebase.js [hunk 2]
```diff
@@ -2,13 +2,14 @@ import firebase from 'firebase/app'
...
+  projectId: process.env.REACT_APP_FIREBASE_PROJECT_ID,
+  storageBucket: process.env.REACT_APP_FIREBASE_STORAGE_BUCKET, 
+  messagingSenderId: process.env.REACT_APP_FIREBASE_MESSAGING_SENDER_ID,
+  appId: process.env.REACT_APP_FIREBASE_APP_ID,
 })
 
-export const auth = app.auth()
-export default app
+const auth = app.auth()
+export { app, auth }
```
Comment (rhen92) [inline src/firebase.js:14]: I like this cleaner refactored export statement!

File: src/firebase.js [hunk 3]
```diff
@@ -2,13 +2,14 @@ import firebase from 'firebase/app'
 import 'firebase/auth'
 
 const app = firebase.initializeApp({
-  apiKey: process.env.REACT_APP_FIREVASE_API_KEY,
-  authDomain: process.env.REACT_APP_FIREVASE_AUTH_DOMAIN,
-  projectId: process.env.REACT_APP_FIREVASE_PROJECT_ID,
-  storageBucket: process.env.REACT_APP_FIREVASE_STORAGE_BUCKET, 
-  messagingSenderId: process.env.REACT_APP_FIREVASE_MESSAGING_SENDER_ID,
-  appId: process.env.REACT_APP_FIREVASE_APP_ID,
+  apiKey: process.env.REACT_APP_FIREBASE_API_KEY,
```
Comment (rhen92) [inline src/firebase.js:5]: Nice catch on the accidental misspelling.

**Comment by rhen92** [comment]
It looks good. We just need to make sure we clean up the commented out code and the console.log on next PR.

**Review by Thee-Dust — APPROVED** [review]
LGTM

**Pull request merged**